---
draft: false
title: Direct characterization of cis-regulatory elements and functional
  dissection of complex genetic associations using HCR-FlowFISH
summary: Effective interpretation of genome function and genetic variation requires a shift from epigenetic mapping of cis-regulatory elements (CREs) to characterization of endogenous function. We developed hybridization chain reaction fluorescence in situ hybridization coupled with flow cytometry (HCRâ€“FlowFISH), a broadly applicable approach to characterize CRISPR-perturbed CREs via accurate quantification of native transcripts, alongside CRISPR activity screen analysis (CASA), a hierarchical Bayesian model to quantify CRE activity. Across >325,000 perturbations, we provide evidence that CREs can regulate multiple genes, skip over the nearest gene and display activating and/or silencing effects. At the cholesterol-level-associated FADS locus, we combine endogenous screens with reporter assays to exhaustively characterize multiple genome-wide association signals, functionally nominate causal variants and, importantly, identify their target genes.
tags:
  - humanevo
publication_types:
  - "2"
authors:
  - Reilly SK
  - Gosai SJ
  - Gutierrez A
  - admin
  - Ulirsch JC
  - Kanai M
  - Mouri K
  - Berenzy D
  - Kales S
  - Butler GM
  - Gladden-Young A
  - Bhuiyan RM
  - Stitzel ML
  - Finucane HK
  - Sabeti PC
  - Tewhey R
doi: 10.1038/s41588-021-00900-4
publication: Nature Genetics 53, 1166-1176
featured: false
image:
  filename: "flowfish.jpg"
  focal_point: ""
  preview_only: true
date: 2021-07-29
external_link: https://www.nature.com/articles/s41588-021-00900-4
---
